src/components/TenantSelector/TenantSelector.tsx

```diff
@@ -149,7 +149,7 @@ export class TenantSelector extends React.Component<
               data-id="tenant-input"
               autoFocus={true}
               onChange={this.onTenantChange}
-              onPressEnter={this.checkTenantValidity}
+              onPressEnter={tenant ? this.checkTenantValidity : undefined}
```

Comment:
Shouldn't we instead make that check inside `checkTenantValidity`?

Reply:
That's where I started, but wound up reverting it for reasons I don't remember. I'll see if I remember why.

Reply:
I moved it to the func